items/weapons/secondary.json

```diff
@@ -75,5 +75,18 @@
 			4,
 			6
 		]
+	},
+	"1938200223": {
+		"name": "P-113 Verdict",
+		"description": "The gas-operated, semiautomatic P-113 chambers the 14mm 'Rapid Deliberation,' the largest centerfire cartridge of any magazine-fed pistol.",
+		"damage": 125,
+		"capacity": 10,
+		"recoil": 40,
+		"fire_rate": 450,
+		"fire_mode": [],
```

Comment:
no fire modes for this one?

Reply:
Resolved in [f3605f2](https://github.com/helldivers-2/json/pull/41/commits/f3605f27973a3c068a3c69e8c4a72f9d4a223a79)